Course_Part_08/src/assingment31/Animal.java

```diff
@@ -0,0 +1,10 @@
+package assingment31;
+
+public interface Animal {
```

Comment:
Voor interfaces is het redundant om `public abstract` voor de methode definitie te zetten, omdat deze altijd public zijn en altijd abstract en kan daarom worden weggelaten.

In tegenstelling tot `abstracte classes`, die kunnen ook "gewone" methodes bevatten en daarom moet er voor een abstract methode het er wel voor staan.